commit 6e30efe570f1ba135747c6d8f4004e78cd24c49c
Author: Florian Weimer <fweimer@redhat.com>
Date:   Wed Mar 12 11:10:14 2025 +0100

    nptl: Include <stdbool.h> in tst-pthread_gettid_np.c
    
    The test uses the while (true) construct.

diff --git a/nptl/tst-pthread_gettid_np.c b/nptl/tst-pthread_gettid_np.c
index 6a98d864e222b9f5..ced42915106f5390 100644
--- a/nptl/tst-pthread_gettid_np.c
+++ b/nptl/tst-pthread_gettid_np.c
@@ -20,6 +20,7 @@
 #include <pthread.h>
 #include <sched.h>
 #include <signal.h>
+#include <stdbool.h>
 #include <support/check.h>
 #include <support/xthread.h>
 #include <unistd.h>
